lib/jsdom/living/nodes/HTMLTextAreaElement-impl.js

```diff
@@ -44,7 +46,12 @@ class HTMLTextAreaElementImpl extends HTMLElementImpl {
     super._childTextContentChangeSteps();
 
     if (this._dirtyValue === false) {
+      const oldValue = this._getValue();
       this._rawValue = childTextContent(this);
+      if (oldValue !== this._getValue()) {
+        this._selectionStart = this._selectionEnd = 0;
+        this._selectionDirection = "none";
+      }
```

Comment:
I don't see this in the spec. Is this change necessary to pass any tests?

Reply:
You are right, this is not needed for these tests.  I have removed this part of the change.